io.catenax.material_demand/RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [1.0.0] - 2023-10-13
```

Comment:
please change the date to today so that we can go for MS3 today in our meeting.

Reply:
done